src/base64encode.js

```diff
@@ -1,9 +0,0 @@
-export default function base64Encode(string) {
-  if (typeof btoa === 'function') {
-    return btoa(string);
-  } else if (typeof Buffer === 'function') {
-    return Buffer.from(string).toString('base64');
-  } else {
-    throw new Error('No native way to base64 encode');
-  }
-}
```

Comment:
removing the need for this removes the need for a polyfill. Could you make sure `Base64` isn't in the package.json?